frontend/src/components/filter/FilterPanel.js

```diff
@@ -1,22 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import { isArray } from 'lodash';
 import FilterMenu from './FilterMenu';
 import FilterPills from './FilterPills';
 import { filterConfigProp, filterProp } from './props';
 
-export default function FilterPanel(props) {
-  const { onRemoveFilter, filters, filterConfig } = props;
+export default function FilterPanel({
+  onRemoveFilter,
+  filters,
+  filterConfig,
+  onApplyFilters,
+  applyButtonAria,
+  allUserRegions,
+}) {
+  const [filtersToShow, setFiltersToShow] = useState([]);
+
+  useEffect(() => {
+    // Determine if filters contain all regions.
+    const passedRegionFilters = filters.filter((f) => f.topic === 'region').map((r) => {
+      if (isArray(r.query)) {
```

Comment:
As you were saying, I think if we actually want the URL to theoretically match from viewer to destination, we will have to include all visible regions in the URL

Reply:
?region.nin[]=1&region.in[]=2

Reply:
I will discuss this with Josh.

Reply:
Region not in has been removed.